src/mod2c_core/units.c

```diff
@@ -851,6 +852,55 @@ static void init()
 	goto l0;
 }
 
+#if USE_LEGACY_UNITS == 0
+/*
+  Convert str to double using atof.
+  Allows consistency with BlueBrain/nmodl
+*/
+
+static double getflt() {
+	int c;
```

Comment:
this code is duplicated and derived from old/legacy implementation but hopefully we will deprecate mod2c repo soon.